Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -0,0 +1,107 @@
+#import "SentryTimeToDisplayTracker.h"
+#import "SentryCurrentDate.h"
+#import "SentryFramesTracker.h"
+#import "SentrySpan.h"
+#import "SentrySpanContext.h"
+#import "SentrySpanId.h"
+#import "SentrySpanOperations.h"
+#import "SentrySwift.h"
+#import "SentryTracer.h"
+
+#if SENTRY_HAS_UIKIT
+
+@interface
+SentryTimeToDisplayTracker () <SentryFramesTrackerListener>
+
+@property (nonatomic, strong) SentrySpan *initialDisplaySpan;
```

Comment:
`h`: As the SentryTracer has a strong reference to this class with the `setFinishCallback`, I think we should use `weak` here to avoid a possible retain cycle. The tracer stays in memory with a strong reference in the `SentryPerformanceTracker` anyways. I don't think we need another `strong` reference here if I'm not mistaken.